modules/library/src/main/scala/zio/elasticsearch/ElasticQuery.scala

```diff
@@ -42,8 +26,9 @@ object ElasticQuery {
    * @return
    *   an instance of [[zio.elasticsearch.query.BoostingQuery]] that represents the boost query to be performed.
    */
+
   final def boosting[S: Schema](
-    negativeBoost: Float,
+    negativeBoost: BoostRange,
```

Comment:
Hmm... Let's revert this. Just use `Float`. Two reasons: 
- I don't like this approach.
- Even boosting is between 0 and 1 in your case, on queries, it actually can be more than 1, I would say.

Please create a follow-up issue to improve boosting.